src/sentry/api/serializers/models/project_filters.py

```diff
@@ -0,0 +1,6 @@
+from typing import TypedDict
+
+
+class ProjectFiltersResponse(TypedDict):
+    id: int
+    active: bool
```

Comment:
I'm not sure whether it's safe to document this as a `bool`. Per the comment on `ProjectFiltersEndpoint.get`:

```
                    # 'active' will be either a boolean or list for the legacy browser filters
                    # all other filters will be boolean
```

Looking at `inbound_filters.get_filter_state`, there's one path where it returns arbitrary `ProjectOption.objects.get_value` results in the `_legacy_browsers_filter` case. Is that worth documenting, and if so how should we capture what seems to be freeform data?